script.js

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable max-classes-per-file */
+document.getElementById('date').innerHTML = Date();
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(title) {
+    const books = Store.getBooks();
+    const index = books.findIndex((book) => book.title === title);
+    if (index !== -1) {
+      books.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(books));
+    }
+  }
+}
```

Comment:
- [ ]  [ OPTIONAL ] when I remove a book and reload the page it comes back please make sure when removing a book should not  display again  after reload

Reply:
please note the MIT link will work after merging the license is only in the main branch for now